### DISCUSSION TITLE
Is there a way we can upload schemal.sql to supabase with an endpoint?

### DISCUSSION THREAD

**Discussion by goldcoders** [supabase/supabase#3653] [Questions]
I dont know if there is an endpoint in supabase.io that i can use to integrate in my build process
so that during first deploy of my app the schema.sql would be posted to supabase.io
I wanna make this seemless for the clients , so they dont need to enter sql.

**Comment by silentworks** [comment]
You can use a migration tool to do this. I currently use one with GitHub Actions to deploy one of my project's.
